src/modules/statement/stmt.rs

```diff
@@ -34,7 +34,10 @@ use crate::modules::function::{
 };
 use crate::modules::imports::import::Import;
 use crate::modules::main::Main;
-use crate::modules::builtin::echo::Echo;
+use crate::modules::builtin::{
+    echo::Echo,
+    cd::Cd
+};
```

Comment:
i've never seen this formatting for imports before in the codebase, i think it should be like this:

```rs
use crate::modules::builtin::echo::Echo;
use crate::modules::builtin::cd::Cd;
```

Reply:
Well right now are just 2 builtin but if we plan to do more the actual solution is more readable

Reply:
> i've never seen this formatting for imports before in the codebase

Afaik this is perfectly fine formatting, @b1ek